rootfs/etc/apache2/sites-enabled/000-default.conf

```diff
@@ -0,0 +1,29 @@
+<VirtualHost *:80>
+	# The ServerName directive sets the request scheme, hostname and port that
+	# the server uses to identify itself. This is used when creating
+	# redirection URLs. In the context of virtual hosts, the ServerName
+	# specifies what hostname must appear in the request's Host: header to
+	# match this virtual host. For the default virtual host (this file) this
+	# value is not decisive as it is used as a last resort host regardless.
+	# However, you must set it for any further virtual host explicitly.
+	#ServerName www.example.com
+
+	ServerAdmin webmaster@localhost
+	DocumentRoot /var/www/owncloud
```

Comment:
Propose to replace `/var/www/owncloud` with a env var we can substitute i.e. `${APACHE_WEBROOT}`.
This would make it easier to test, if the workspace in drone is not located at `/var/www/owncloud` - like it is right now in core. 

We could create a wrapper around apachectl / or work with a custom entrypoint - thoughts/recommendations @tboerger

Reply:
Since drone overwrites at some conditions the entrypoint I would say let's use an Apachectl wrapper